c2rust-transpile/src/lib.rs

```diff
@@ -218,9 +219,32 @@ fn get_module_name(
     file.to_str().map(String::from)
 }
 
+fn target_arch_from_args(clang_args: &[& str]) -> Option<String> {
```

Comment:
Clang knows the target for the original source file, so couldn't we export that from `AstExporter` instead?

Reply:
Good call, done.